ddtrace/tracer/tracer_test.go

```diff
@@ -1950,9 +1961,23 @@ loop:
 			time.Sleep(time.Millisecond)
 		}
 	}
+	as := &aggregableSpan{
+		key: aggregation{
+			Name: "http.request",
+		},
+		// Start must be older than latest bucket to get flushed
+		Start:    time.Now().UnixNano() - 3*500000,
```

Comment:
Any particular reason to choose `3*500000`?

Reply:
Because the bucketSize for the stats concentrator here is set to `500000`, setting the start time for the span to be `time.Now().UnixNano() - 3*500000` ensures that this span does not belong to the current bucket (which does not get flushed when `t.stats.flushAndSend(time.Now(), withoutCurrentBucket)` is called in `tracer.go`). If the start time for the span is set to `time.Now().UnixNano() - 250000`, for example, this test case will fail.

I chose the bucketSize of `500000` copying these [unit tests ](https://github.com/DataDog/dd-trace-go/blob/122edbe543ca338f3a6b64a0395e124bccd124ea/ddtrace/tracer/stats_test.go#L125) in `stats_test.go`

To increase code clarity I can set some variable `bucketSize` to 500000 and use that throughout!